### ISSUE TITLE
Assets css (admin/components/...)

### ISSUE THREAD

**Issue by enkaho** [symfony2admingenerator/GeneratorBundle#159]
Hi, 
Thx for this awesome generator ! But it's hard to follow all the change without an up to date doc...
How to install all css for adminLTE and bootstrap components ?

When i do :  
```
 php app/console assetic:dump
```
> [file+]  ../web/assetic/css/compiled/admingenerator.css
>
> [RuntimeException]
> The source file  ../web/admin/components/bootstrap/dist/css/bootstrap.css" does not exist.  

And if i follow the installation via composer doc , the dependencies with cedriclombardot/admingenerator-generator-bundle are broken ...
> Problem 1
>    - Conclusion: don't install cedriclombardot/admingenerator-generator-bundle v1.1.3
>    - Conclusion: don't install cedriclombardot/admingenerator-generator-bundle v1.1.2
>    - Conclusion: don't install cedriclombardot/admingenerator-generator-bundle v1.1.1
>    - Conclusion: remove twig/extensions v1.2.0
>    - Installation request for cedriclombardot/admingenerator-generator-bundle ~1.1 -> satisfiable by >cedriclombardot/admingenerator-generator-bundle[v1.1.0, v1.1.1, v1.1.2, v1.1.3].
>    - Conclusion: don't install twig/extensions v1.2.0
>    - cedriclombardot/admingenerator-generator-bundle v1.1.0 requires twig/extensions 1.0.* -> 
> satisfiable by twig/extensions[v1.0.0, v1.0.1].
>    - Can only install one of: twig/extensions[v1.0.0, v1.2.0].
>   - Can only install one of: twig/extensions[v1.0.1, v1.2.0].
>   - Installation request for twig/extensions == 1.2.0.0 -> satisfiable by twig/extensions[v1.2]

I'm on SF 2.7.1.
```
   "require": {
        "php": ">=5.3.3",
        "symfony/symfony": "~2.7@beta",
        "doctrine/orm": "~2.2,>=2.2.3,<2.5",
        "doctrine/dbal": "<2.5",
        "doctrine/doctrine-bundle": "~1.4",
        "symfony/assetic-bundle": "~2.3",
        "symfony/swiftmailer-bundle": "~2.3",
        "symfony/monolog-bundle": "~2.4",
        "sensio/distribution-bundle": "~3.0,>=3.0.12",
        "sensio/framework-extra-bundle": "~3.0,>=3.0.2",
        "incenteev/composer-parameter-handler": "~2.0",
 
         "friendsofsymfony/user-bundle": "^1.3",   
         "excelwebzone/recaptcha-bundle": "dev-master",  
         "jms/serializer-bundle": "0.13.*@dev",
         "stof/doctrine-extensions-bundle": "dev-master", 
        "symfony2admingenerator/generator-bundle": "^2.0@beta",
        "vich/uploader-bundle": "^0.14.0",
        "symfony2admingenerator/form-extensions-bundle": "dev-master", 
 
    },

```
Thx for your help !

**Comment by enkaho** [comment]
fixed with  
```
php app/console admin:assets-install
```

**Comment by sescandell** [comment]
Hi,

Yes we know that we lack of documentation. Someone new in my team just install this bundle and have been quickly blocked because of missing documentation. This is probably one of the biggest challenge we have to handle to make this bundle easy to use...

Any help for that is welcome :)
